docs/guides/add-new-provider.md

```diff
@@ -221,6 +235,30 @@ _e.g. `https://api.example.com/?accessKey=<apikey>`_
 - `in` must be set to `query`
 - `name` is the query param name that holds the API key (e.g. `accessKey`)
 
+## Integration parameters
+
+Define list of integration parameters specific for the service provider. Examples of integration parameters: region where is API deployed or API instance ID.
```

Comment:
```suggestion
Optionally define a list of integration parameters specific for the service provider. Integration parameter can be, for example, region where the API is deployed or an API instance ID.
```

However this seems somewhat redundant with the explanation above and a glossary entr for integration parameter.

Reply:
I omitted redundant description and added link to glossary.